blocks/footer/footer.css

```diff
@@ -1,14 +1,92 @@
 footer {
-  padding: 2rem;
-  background-color: var(--overlay-background-color);
+  background-color: #3e40a0;
   font-size: var(--body-font-size-s);
+  color: white;
 }
 
-footer .footer {
+.footer {
   max-width: 1200px;
   margin: auto;
 }
 
-footer .footer p {
-  margin: 0;
-}
\ No newline at end of file
+
+.footer ul {
+  list-style-type: none;
+}
+
+.footer a {
+  color: white;
+  text-decoration: none;
+}
+
+/* main footer section that contains the different subsections */
+footer > div > div {
+  display: flex;
+  flex-flow: row wrap;
+  padding: 16px;
+}
+
+/* each footer subsection */
+footer > div > div > div {
+  min-width: 50%;
+}
+
+/* links subsection */
+footer > div > div > div:first-child {
+  display: flex;
+  flex-flow: row wrap;
+
+}
+
+/* each link group */
+footer > div > div > div:first-child > ul {
+  padding-inline-start: 0;
+  padding-inline-end: 50px;
+}
+
+@media(max-width: 600px) {
+  footer > div > div > div:first-child > ul {
+    min-width: 100%;
+  }
+}
+
+@media(min-width: 600px) and (max-width: 900px) {
+  footer > div > div > div:first-child > ul {
+    min-width: 40%;
+  }
+}
+
+/* title of each link section */
+footer > div > div > div:first-child > ul > li:first-child {
+  font-weight: bold;
+}
+
+/* netcentric logo */
+footer > div > div > div:nth-child(2) > p > a > picture > img {
+  width: 150px;
+  height: 40px;
+}
+
+/* social subsection */
+footer > div > div > div:nth-child(3) > ul {
+  display: inline-flex;
+  float: right;
+  padding-inline-start: 0;
+  font-size: large;
+}
+
+.icon {
```

Comment:
FYI @alexcarol this affects other blocks too (eg. its breaking the header on my branch after merging). So please always prefix with the block, ie. `footer .icon`.

But don't worry about changing it here, I will do it on the header branch.